rbac/management/management/commands/sync_schemas.py

```diff
@@ -73,6 +73,7 @@ def copy_custom_roles_to_public(self, tenant):
                     if not access.tenant:
                         access.tenant = tenant
                     access.role = role
+                    access.permission = Permission.objects.get(permission=access.permission.permission)
```

Comment:
This makes me wonder the line 82 might get into trouble since it is querying within public schema. Where actually we would like to query within tenant schema.

And do we have to do similar thing as line 73-74 for other resources like group, policy, etc.?

Reply:
The Permissions should be seeded into the public schema though, right? So we'll want to use those rather than the ones in the given tenant schemas, correct?

And yes, each other object/model has a check on its tenant before we move to the public schema.